app/models/shipit/stack.rb

```diff
@@ -251,8 +251,28 @@ def lock_reverted_commits!
       end
     end
 
-    def undeployed_commits
+    def active_commits
+      return [] unless active_task?
+
+      scope = commits.reachable.until(active_task.until_commit)
+      scope = if active_task.since_commit == active_task.until_commit
```

Comment:
reassigning to the same makes it a little bit hard to understand what each state holds.
I would change the name to communicate it better.

Reply:
very nitpicky feel free to ignore